helpers.go

```diff
@@ -1,14 +1,22 @@
 package ssz
 
+import "bytes"
+
 // Given ordered objects of the same basic type, serialize them, pack them into BYTES_PER_CHUNK-byte
 // chunks, right-pad the last chunk with zero bytes, and return the chunks.
 // Basic types are either bool, or uintN where N = {8, 16, 32, 64, 128, 256}.
 //
 // Important: due to limitations in Go generics, we will assume the input is already
 // a list of SSZ-encoded objects of the same type.
 func pack(serializedItems [][]byte) ([][]byte, error) {
+	areAllEmpty := true
+	for _, item := range serializedItems {
+		if !bytes.Equal(item, []byte{}) {
+			areAllEmpty = false
```

Comment:
break?

Reply:
This tests if all are empty, not that is any empty

Reply:
Please add the break. As soon as you find one that is not empty, you can stop checking.